# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1521785844 25200
#      Thu Mar 22 23:17:24 2018 -0700
# Branch stable
# Node ID 8b50eab9689eda6d485dd90f8092e57956a32c0e
# Parent  c16ad80274c9abc702963f821047036cfb43cd48
system.tst: fix system call test that fails in corner cases

* system.tst: Adjust test of getpgrp to allow return value to be zero,
for example if Octave runs as the primary process in a container.

diff --git a/test/system.tst b/test/system.tst
--- a/test/system.tst
+++ b/test/system.tst
@@ -218,17 +218,17 @@
 %! x = getpwuid (getuid ());
 %! assert (x.dir, tilde_expand (sprintf ("~%s", x.name)));
 %! assert ("foobar", tilde_expand ("foobar"));
 
 %!error <Invalid call to tilde_expand> tilde_expand ()
 %!error <Invalid call to tilde_expand> tilde_expand ("str", 2)
 
 %!testif HAVE_GETPGRP
-%! assert (getpgrp () > 0);
+%! assert (getpgrp () >= 0);
 
 %!error <... getpgrp> getpgrp (1)
 
 %!assert (getpid () > 0)
 
 %!error <... getpid> getpid (1)
 
 %!testif HAVE_GETPPID
